app/models/votable_ip.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+# Track IP address of anonymous votable voters ('likes' feature)
```

Comment:
This comment suggests an alternative (more intention revealing?) name for this model instead of `VotableIP` would be something like `Voter`, `AnonVoter`, `Guest`, `GuestUser`, etc.

Reply:
Yeah... my place of work made me strangely reluctant to tie the keyword 'Voter' into a 'likes' feature too tightly, it feels like too 'high value' a generic term for it to be used here :) I did have AnonVoter for a while, but same problem (albeit less so) really.

Also a slight niggle that two votes from an IP may not be from the same voter. I had VotingIP briefly but that's so close to using the gem name that I decided to do that in the end.